practice/4_python_part_3/test_task_4.py

```diff
@@ -0,0 +1,26 @@
+import unittest
+from unittest.mock import Mock, patch
+from task_4 import print_name_address
+
+class TestPrintNameAddress(unittest.TestCase):
```

Comment:
lets do some more tests. You could patch `sys.stdout` and check what your program prints out
- define some test cases:
{"case1": {'NUMBER': 2, '--fake-address': 'address', '--some_name': 'name'},
                  "case2": {'NUMBER': 3, '--fake_company': 'company', '--fake-color': 'color'}}
- test
  - if your program returns a dictionary
  - if the dict contains expected keys